build-tools/src/main/resources/software/amazon/awssdk/spotbugs-suppressions.xml

```diff
@@ -293,6 +293,11 @@
         <Bug pattern="NP_BOOLEAN_RETURN_NULL"/>
     </Match>
 
+    <Match>
```

Comment:
No, unfortunately it's not possible because the builder constructor already exists and is used within Amazon. It's weird how that code made it through spot bugs (it was also me btw).